frontend/src/metabase/parameters/components/ParameterSettings/ParameterSettings.tsx

```diff
@@ -132,8 +137,13 @@ const ParameterSettings = ({
           setValue={onChangeDefaultValue}
         />
 
-        <SettingRequiredContainer>
+        <SettingRequiredContainer
+          // This forces re-rendering when switching between parameters,
+          // so that toggles don't slide, which is confusing.
```

Comment:
I see what you mean, but technically this doesn't just make it re-render, it already does that just because the parameter changes. What it does is it makes react throw away the old component a create a new one.
```suggestion
          // This forces the toggle to be a new instance when the parameter changes,
          // so that toggles don't slide, which is confusing.
```